tests/infra_test.go

```diff
@@ -1052,8 +1058,81 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 				Skip("Skip testing with node-labeller, because there are no nodes with kvm")
 			}
 		})
+		AfterEach(func() {
```

Comment:
In utils.go there exists a `CleanNodes` function. Consider extending it to remove this annotation. It already removes some labels.

Reply:
that should be pretty straight forward. 

this loop in CleanNodes

```
                for k := range node.Labels {
                        if strings.HasPrefix(k, "tests.kubevirt.io") {
                                found = true
                                delete(new.Labels, k)
                        }
                }
```
becomes this.

```
                for k := range node.Labels {
                        if strings.HasPrefix(k, "tests.kubevirt.io") || k == nonExistingCPUModelLabel {
                                found = true
                                delete(new.Labels, k)
                        }
                }

                for k := range node.Annotations {
                        if k == v1.LabellerSkipNodeAnnotation {
                                found = true
                                delete(new.Annotations, k)
                        }
                }
```

Reply:
Issue is, that I need to have that clean up code in afterTest, not AfterSuite, because otherwise I would have to update all labeller tests.

Reply:
most test `Description` scoped BeforeTest functions call the `tests.BeforeTestCleanup()` function which then calls CleanNodes().

I'm fine with your logic being in an explicit AfterTest function that's directly associated with the test case that executes it. This isn't generic functionality like the cordon/uncordon logic that the CleanNodes is responsible for.